### ISSUE TITLE
[BACK-END] BDD: File Size Limit for STRs Needs to Conditionally Allow for Larger PDFs

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#16004]
## Acceptance Criteria
- [x] Allow PDFs up to 150mb and all other files up to 50mb for all 526 uploads (including Claim Status Tool)

## Notes
During UAT, we had a service member with a large PDF (95 MB). Although EVSS is going to do the splitting of the file, our back end has work to do for making sure we allow these file sizes to go through. This story is for our end of the code (separate story for the proxy updates that were part of the previous limitation expansion work).

**Comment by luke-gcio** [comment]
Still waiting on FE fix and will close with both.

**Comment by annaswims** [comment]
I went ahead and merged the BE changes.  until the FE changes are in place the FE will limit to 50 mb.

**Comment by annaswims** [comment]
@luke-gcio this ticket is assigned to me, but is there a separate FE ticket? BE work is complete

**Comment by edmangimelli** [comment]
BE work is complete. The FE ticket is #16003
